Watchman.Discord/Areas/Users/Services/RolesService.cs

```diff
@@ -18,23 +18,47 @@ public RolesService(UsersService usersService, UsersRolesService usersRolesServi
             _usersRolesService = usersRolesService;
         }
 
-        public void AddRoleToUser(IEnumerable<Role> safeRoles, MessagesService messagesService, Contexts contexts, string commandRole)
+#if DEBUG
```

Comment:
czemu if debug?

Reply:
Ponieważ jest jedna funkcja która używa, i prawdopodobnie będzie używać tej metody. I jest ona przypisana do komendy "!admin", działa ona tylko w "if debug". Oceniłem że została wykonana tak statycznie, w celu testowania, i uznałem że w tym celu będzie zrobię oddzielną metodę w której będzie można dodać po prostu w trybie debugowania prostym sposobem rolę.

Reply:
ogólnie !admin bym wywalił bo jest niebezpieczne

ktoś przypadkiem wrzuci zmieni ifa, wejdzie to na produkcje, ktoś sie dowie bo to open source
i będziemy mieć problem, bo każdy będzie mógł nadać sobie uprawnienia admina